NEWS

```diff
@@ -9,6 +9,14 @@ Next
 Changes
 -------
 
+* ``AsynchronousDeferredRunTest`` now has ``suppress_twisted_logging`` and
+  ``store_twisted_logs`` parameters that can be used to override the default
+  logging behaviour.  (Jonathan Lange, #942785)
+
+* New fixture ``CaptureTwistedLogs`` that can be used with
+  ``AsynchronousDeferredRunTest`` to attach a detail containing everything
```

Comment:
If I'm not mistaken, CaptureTwistedLogs can be used with any test runner? If so, suggest removing 'with ``Async...``' as it sort of implies that it's only useful in that case. But perhaps I've misunderstood....